includes/admin/class.llms.admin.menus.php

```diff
@@ -195,7 +196,8 @@ public function display_admin_menu() {
 
 		$menu[51] = array( '', 'read', 'llms-separator', '', 'wp-menu-separator' );
 
-		add_menu_page( 'lifterlms', 'LifterLMS', 'read', 'lifterlms', '__return_empty_string', plugin_dir_url( LLMS_PLUGIN_FILE ) . 'assets/images/lifterlms-icon.png', 51 );
+		$icon_url = 'data:image/svg+xml;base64,' . base64_encode( file_get_contents( LLMS_PLUGIN_DIR . 'assets/images/lifterlms-icon-grey.svg' ) );
```

Comment:
Ignore wp phpcs warnings here

```suggestion
		$icon_url = 'data:image/svg+xml;base64,' . base64_encode( file_get_contents( LLMS_PLUGIN_DIR . 'assets/images/lifterlms-icon-grey.svg' ) ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents -- Not a remote URL.
```
```suggestion
		$icon_url = 'data:image/svg+xml;base64,' . base64_encode( file_get_contents( LLMS_PLUGIN_DIR . 'assets/images/lifterlms-icon-grey.svg' ) );
```

Reply:
When I ran `composer check-cs-errors`, it didn't give me any warnings. In fact, even if a remove a bunch of exclusions from `phpcs.xml`, I don't get any errors. Looks like something is wrong with my development environment.